modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20817,6 +20837,7 @@ yearly-x-times=Yearly, {0} Times
 years=Years
 yes=Yes
 yes-cancel=Yes, cancel.
+yes-change-unit-of-measure=Yes, change UoM
```

Comment:
The key looks wrong, it should be `yes-change-uom` and missing a dot at the end since it's a sentence.